src/modules/drawing/components/DrawingToolbox.vue

```diff
@@ -78,15 +79,11 @@ onMounted(() => {
     }
 })
 
-watch(drawingName, () => {
-    sanitizeDrawingName()
-    debounceSaveDrawing({
-        // Lowering the risk of the user changing the name and leaving the drawing module without the name being saved.
-        // It is terrible design that this can occur, but moving the drawing name management other places raises the complexity tenfold
-        // so that is a trade-off we can live with.
-        debounceTime: 500,
-        drawingName: drawingName.value.trim(),
-    })
+watch(drawingName, (newName) => {
```

Comment:
I was once told that one should avoid to have too many `watch`. The reasoning was some performance issues, I don't know if that's true, but maybe it's also because of unintended side effects or something?

Anyway, personally I think this here could be solved like this:
```js
const editDrawing = computed({
  get() {
    return store.state.drawing.name
  },
  set(value) {
    debounceSaveDrawingName(value)
  }
})

```

I guess one argument could be that like this you have it in one location and don't have to copy values around?

Reply:
Very nice!